### PR TITLE
Allow users to choose which database to use from the frontend

### PR DESCRIPTION
**Pull request by Lev-Stambler**
- Add a modal library component
- Integrate it with the search

### CODE REVIEW

File: apps/api-frontend/src/app/pages/search-pages/search.css [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+
+.app .search .search-options {
+  display: grid;
+  width: 100%;
+  align-items: center;
+  justify-content: left;
+  padding: 0.5rem 1rem;
+  border: 1px dotted black;
+  margin: 1rem 0;
+  border-radius: 0.5rem;
```
Comment (vitashubin) [inline apps/api-frontend/src/app/pages/search-pages/search.css:8]: nit: styleguide prefers to alphabetize the properties - I'll leave that up to you, but do highly recommend it (it comes in especially handy as apps grow)

File: libs/components/src/lib/modal-chooser-db/modal-chooser.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    width: 100%;
+    grid-template-columns: 4fr 1fr;
+    gap: 1rem;
+    padding: 1rem;
+    justify-content: center;
+    align-items: top;
+}
+
+.search-bar-container button {
+    height: 30px;
```
Comment (vitashubin) [inline libs/components/src/lib/modal-chooser-db/modal-chooser.css]: nit: stay consistent with using rem

**Review by vitashubin — CHANGES_REQUESTED** [review]
Can you add a link to the site? :) Thanks!

**Comment by Lev-Stambler** [comment]
> Can you add a link to the site? :) Thanks!

I cannot deploy the frontend until it merges into master via Netlify. Here are images though

**Comment by Lev-Stambler** [comment]
Screenshot_2020-07-13_135253
Screenshot_2020-07-13_135337

File: libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    props.onClose(db);
+  }
+
+  return (
+    <div>
+      <button onClick={openModal}>Select a scholarly database</button>
+      <Modal
+        isOpen={modalIsOpen}
+        onRequestClose={closeModal}
+        style={customStyles}
```
Comment (vitashubin) [inline libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx]: prefer to have this Modal have an attached css class, and the style be defined in a .css file

File: libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        contentLabel="Example Modal"
```
Comment (vitashubin) [inline libs/components/src/lib/modal-chooser-db/modal-chooser-db.tsx]: update?

File: apps/api-frontend/src/app/pages/search-pages/search.css (hunk 1 above)
Comment (Lev-Stambler) [reply apps/api-frontend/src/app/pages/search-pages/search.css:8]: Done (ran through the alphabet a few times in my head :) )

**Review by vitashubin — CHANGES_REQUESTED** [review]
> > Can you add a link to the site? :) Thanks!
> 
> I cannot deploy the frontend until it merges into master via Netlify. Here are images though

Thanks for the images! Mind adding one more for what it looks like when an option is selected?

**Comment by vitashubin** [comment]
> > > Can you add a link to the site? :) Thanks!
> > 
> > 
> > I cannot deploy the frontend until it merges into master via Netlify. Here are images though
> 
> Thanks for the images! Mind adding one more for what it looks like when an option is selected?

Quick note - there's a chrome extension called screencast that lets you take videos of what's happening on your screen and link them to people, might be helpful to have in these situations

**Pull request merged**